packages/affiliates/libs/affiliates/devmining.js

```diff
@@ -300,7 +306,7 @@ module.exports = ({ queries, empAbi, coingecko, synthPrices, firstEmpDate }) =>
         collateralTokens,
         async address => await getCoingeckoPriceHistory(address, "usd", startTime, endTime)
       ),
-      Promise.map(
+      Promise.mapSeries(
```

Comment:
this was a change when trying to understand why i was rate limited. it would slow down the calls slightly, executing requests one at a time vs all at once. I left it in after realizing it was an issue with the api key, because its probably better to call the api slower anyway.